### DISCUSSION TITLE
Spectrum on ESP32?

### DISCUSSION THREAD

**Discussion by stevencarlover** [PlummersSoftwareLLC/NightDriverStrip#246] [General]
Hello!

I've been a big fan of Dave's Garage for some time decided recently I would like to build the Spectrum Analyzer.

I noticed the ESP32 is not a selectable device for the spectrum project, is this a HW limitation of ESP32 (RAM) or is this something that would need to be configured manually in platformio.ini?

Also I was wondering the same question for the ESP32-S3-WROOM-1? I actually have an ESP32-S3-BOX with this chipset I was hoping to run spectrum on as well? 

I have tried modifying the configs to run on both chips with no luck as of yet.

Just wondering if there is a HW reason why this can't be done and if I am wasting my time.

Thanks!,
Steve

**Comment by davepl** [comment]
The spectrum project assumes a particular module, the M5StickCPlus, which includes a screen, microphone, and everything you need.   It uses the ESP32, but has more to it.

Grab one off Amazon, they’re around $30 and well worth it!

https://amzn.to/3TLYIsO￼
MakerFocus M5Stack M5StickC Plus ESP32-PICO-D4 Mini IoT Development Kit WiFi Support Bigger Screen IoT Controller for UIFlow, MicroPython and Ar duino
amzn.to

Cheers,
Dave

**Reply by stevencarlover** [reply]
Hi Dave,

Thanks for the immediate response!

I actually placed the order this morning for the M5StickCPlus,  just a bit eager to get it running so I wanted to try it on HW I had...

Should be coming tomorrow. 

Thanks!,
Steve
